test/ck_tile/smoothquant/test_smoothquant.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier:  MIT
+
+#include "test_smoothquant_types.hpp"
+#include "test_smoothquant_util.hpp"
+#include "gtest/gtest.h"
+
+#define TEST_SUITE_NAME TestCkTileSmoothquant
+
+TYPED_TEST_SUITE(TestCkTileSmoothquant, KernelTypesSmoothquant);
```

Comment:
The previous test setup had separated bf16 from fp16 tests. I think it's a good idea here to see what impact combining them has on the build time. If it takes more than a minute or two in the new build, I think I would recommend separating them into multiple .cpps so they can be built in parallel.